src/app/pages/common/ix-forms/services/ix-formatter.service.ts

```diff
@@ -0,0 +1,183 @@
+export class IxFormatter {
+  static humanReadable: string;
+  static readonly IECUnits = ['B', 'KiB', 'MiB', 'GiB', 'TiB', 'PiB'];
+
+  static readonly memorySizeFormatting: (val: string | number) => string = (value: string | number) => {
+    value = value.toString();
+    if (!value) {
+      return '';
+    }
+    let formatted = '';
+    const memoryInNumbers = IxFormatter.convertHumanStringToNum(value);
```

Comment:
Why do we need to convert humanStringToNum here when we don't expect value to be a "human" string?